service/common/src/main/java/org/apache/polaris/service/events/listeners/PolarisPersistenceEventListener.java

```diff
@@ -92,13 +98,14 @@ public void onAfterTableCreated(AfterTableCreatedEvent event, CallContext callCt
         org.apache.polaris.core.entity.PolarisEvent polarisEvent = new org.apache.polaris.core.entity.PolarisEvent(
                 event.getEventId(),
                 event.getRequestId(),
-                event.getEventCount(),
+                event.getClass().getSimpleName(),
                 event.getTimestampMs(),
-                event.getActor(),
-                event.getIcebergOperationType(),
-                event.getPolarisCustomOperationType(),
+                event.getUser(),
                 event.getResourceType(),
                 event.getTableIdentifier().toString());
+        Map<String, String> additionalParameters = Map.of("table-uuid", event.getTableMetadata().uuid(), "metadata", TableMetadataParser.toJson(event.getTableMetadata()));
```

Comment:
The whole metadata is probably too big to put in some cases

Reply:
Yes, we do need to handle this. I'm not sure I want to handle it here yet. Thinking of handling this later when we finish RegisterTable